app.rb

```diff
@@ -0,0 +1,52 @@
+require 'sinatra/base'
+require 'sinatra/reloader'
+require 'capybara'
+require './lib/peep'
+require './lib/user'
+require './database_connection_setup'
+
+class Chitter < Sinatra::Base
+  enable :sessions
+  configure :development do
+    register Sinatra::Reloader
+  end
+
+  get '/' do
+    @user = User.find(id: session[:user_id])
+    @peeps = Peep.all_peeps
+    erb(:index)
+  end
+
+  get '/new-peep' do
+    erb(:new_peep)
+  end
+
+  post '/post' do
+    Peep.create(message: params[:message])
+    redirect '/'
+  end
+
+  get '/sign-up' do
+    erb(:sign_up)
+  end
+
+  post '/new-account' do
```

Comment:
I like the way you broke this out into separate lines to avoid making one really long one, It looks far neater.